src/modules/organisation/api/organisation.controller.spec.ts

```diff
@@ -96,4 +99,51 @@ describe('OrganisationController', () => {
             expect(organisationUcMock.findOrganisationById).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe('findOrganizations', () => {
```

Comment:
When clause is missing (test structure).

Reply:
✅